cache/src/test/kotlin/com/dropbox/android/external/cache4/TestClock.kt

```diff
@@ -1,8 +1,12 @@
 package com.dropbox.android.external.cache4
 
+import kotlin.time.Duration
+import kotlin.time.ExperimentalTime
+
+@ExperimentalTime
 class TestClock(
```

Comment:
I wonder if we can replace the custom `Clock` with `TimeSource`, `TimeSource.Monotonic`, `TestTimeSource` from kotlin.time?